src/schematools/contrib/django/management/commands/create_views.py

```diff
@@ -0,0 +1,66 @@
+import re
+from collections import defaultdict
+from typing import Iterable, List, Optional
+
+from django.core.management import BaseCommand, CommandError
+from django.db import DatabaseError, connection, router, transaction
+
+from schematools.contrib.django.factories import schema_models_factory
+from schematools.contrib.django.models import Dataset
+
+
+class Command(BaseCommand):
+    help = "Create the views based on the uploaded Amsterdam schema's."
+    requires_system_checks = []  # don't test URLs (which create models)
+
+    def handle(self, *args, **options):
+        create_views(self, Dataset.objects.db_enabled())
+
+
+def create_views(
+    command: BaseCommand,
+    datasets: Iterable[Dataset],
+    base_app_name: Optional[str] = None,
+) -> None:  # noqa: C901
+    """Create tables for all updated datasets.
+    This is a separate function to allow easy reuse.
+    """
+    errors = 0
+    command.stdout.write("Creating tables")
```

Comment:
Shouldn't the message be "creating views"?

Reply:
Fixing